src/main/resources/pdiresources/jobs/validations.kjb

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
and just to be consistent with our naming: "validations-job.kjb"

or we could also just remove the "-job" part since it is clear they are jobs...up to you

Reply:
let us remove it